src/tkey-stats.ts

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import memoizee from 'memoizee';
+
+const CONTRACT = 'KT1VaEsVNiBoA56eToEK6n6BcPgh1tdx9eXi';
+const BURN_ADDRESS = 'tz1burnburnburnburnburnburnburjAYjjX';
+const DECIMALS = 18n;
+const TOTAL_SUPPLY = 14_000_000_000_000_000_000_000_000n;
+const TOTAL_SUPPLY_WITH_DECIMALS = TOTAL_SUPPLY / 10n ** DECIMALS;
+
+const INCENTIVES = 1_000_000n;
+const INVESTMENT = 1_000_000n;
+const DEVELOPER_REWARDS = 2_000_000n;
```

Comment:
I suppose these values are not constant, so they should be fetched from TZKT too, see the balance of `tz1bxHEHAtKubVy44vBDFVEZ1iqYPYdJVS9U` here: https://tzkt.io/KT1VaEsVNiBoA56eToEK6n6BcPgh1tdx9eXi/tokens/0/holders

Reply:
Things didn't change for a while. But that's a good idea to make it dynamic